packages/contracts-ops/contracts/test/RebootBridgeRouter.t.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "forge-std/Test.sol";
+import {RebootTest} from "./Reboot.t.sol";
+import {UpgradeBeacon} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeacon.sol";
+import {NomadTest} from "@nomad-xyz/contracts-core/contracts/test/utils/NomadTest.sol";
+import {BridgeTestFixture} from "@nomad-xyz/contracts-bridge/contracts/test/utils/BridgeTest.sol";
+import {BridgeRouterBaseTest} from "@nomad-xyz/contracts-bridge/contracts/test/BridgeRouterBase.t.sol";
+import {BridgeRouterTest} from "@nomad-xyz/contracts-bridge/contracts/test/BridgeRouter.t.sol";
+import {BridgeRouterHarness} from "@nomad-xyz/contracts-bridge/contracts/test/harness/BridgeRouterHarness.sol";
+import {TokenRegistryHarness} from "@nomad-xyz/contracts-bridge/contracts/test/harness/TokenRegistryHarness.sol";
+import {IBridgeRouterHarness} from "@nomad-xyz/contracts-bridge/contracts/test/harness/IBridgeRouterHarness.sol";
+import {BridgeToken} from "@nomad-xyz/contracts-bridge/contracts/BridgeToken.sol";
+import {TokenRegistry} from "@nomad-xyz/contracts-bridge/contracts/TokenRegistry.sol";
+
+contract BridgeRouterRebootTest is RebootTest, BridgeRouterTest {
+    address bridgeRouterHarnessImpl;
+    address tokenRegistryHarnessImpl;
+
+    string constant ethereum = "ethereum";
```

Comment:
Why are we hardcoding the domain?  I assume that RebootEthereumBridgeRouter would test this

Reply:
all fork tests are run against ethereum rpc right now